setup.py

```diff
@@ -189,7 +196,16 @@ def cython_extensions(package_names, module_names, include_dirs, libraries,
 TEST_LOADER = 'tests:Loader'
 TEST_RUNNER = 'tests:Runner'
 
-PACKAGE_DATA = {}
+PACKAGE_DATA = {
+    'grpc._adapter': [
+        'credentials/roots.pem'
+    ],
+    'grpc._cython': [
+        '_windows/grpc.def',
+        '_windows/grpc_c.32.python',
+        '_windows/grpc_c.64.python',
```

Comment:
That's fine. Trying to understand where we're going and what will be left afterward.